src/components/network-chart/index.js

```diff
@@ -196,8 +196,8 @@ function _getVirtualListItems(props: Props): NetworkChartRowProps[] {
       };
     }
     // Compute the positioning of the network markers.
-    const startPosition = _timeToCssPixels(props, networkPayload.startTime);
-    const endPosition = _timeToCssPixels(props, networkPayload.endTime);
+    const startPosition = _timeToCssPixels(props, marker.start);
+    const endPosition = _timeToCssPixels(props, marker.start + marker.dur);
```

Comment:
I think that all our marker components deal with marker.start/dur instead of looking at the payload, at least for the general things.